pkgs/by-name/te/teamspeak-pluginsdk/package.nix

```diff
@@ -0,0 +1,42 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  teamspeak_client,
+
+  version ? "26",
+  hash ? "sha256-oXyOqje4x79ymkG3YoS4zb25R2+HaYiUDwpZnUErag4=",
+}:
+
+stdenv.mkDerivation rec {
+  pname = "ts3client-pluginsdk";
+  inherit version;
+
+  src = fetchFromGitHub {
+    owner = "teamspeak";
+    repo = pname;
+    rev = version;
+    inherit hash;
+  };
+
+  outputs = [ "dev" "test_plugin" "out" ];
+
+  installPhase = ''
+    runHook preInstall
+
+    mkdir -p $dev $out
```

Comment:
This leaves out as an empty dir which isn't great but you must have an out unfortunately. I tried symlinking out to dev but that results in a cycle as dev refers to out as a propagatedBuildInput.

I'll squash the init of this later.